pkg/operator/staticpod/controller/node/node_controller_test.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"github.com/openshift/library-go/pkg/operator/v1helpers"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: we use to call this `corev1` :-) also this alias is redundant

Reply:
will change to `corev1`